.github/workflows/sub_buildWindows.yml

```diff
@@ -134,3 +134,11 @@ jobs:
           name: ${{ inputs.artifactBasename }}-Logs
           path: |
             ${{ env.logdir }}
+
+      - name: Upload Freecad
+        if: always()
+        uses: actions/upload-artifact@v4
+        with:
+          name: ${{ inputs.artifactBasename }}-Binary
+          path: |
+            ${{ env.builddir }}
```

Comment:
I will remove before finishing the pr, it is just for me so I can directly test it on windows

Reply:
Removed